core/src/components/accordion/test/accordion.spec.ts

```diff
@@ -163,3 +163,24 @@ it('should set default values if not provided', async () => {
 
   expect(accordion.classList.contains('accordion-collapsed')).toEqual(false);
 });
+
+// Verifies fix for https://github.com/ionic-team/ionic-framework/issues/27047
+it('should not animate when false', async () => {
+  const page = await newSpecPage({
+    components: [Item, Accordion, AccordionGroup],
+    html: `
+      <ion-accordion-group animated="false">
+        <ion-accordion>
+          <ion-item slot="header">Label</ion-item>
+          <div slot="content">Content</div>
+        </ion-accordion>
+      </ion-accordion-group>
+    `,
+  });
+
+  const accordionGroup = page.body.querySelector('ion-accordion-group');
```

Comment:
```suggestion
  const accordionGroup = page.body.querySelector('ion-accordion-group')!;
```

Accordion Group is always defined, so we can add this and then remove the `?`